src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -186,19 +170,42 @@ def get_display(my):
         files_div.add_style("max-height: 300px")
         files_div.add_style("height: 300px")
         files_div.add_style("overflow-y: auto")
-        files_div.add_border()
         files_div.add_style("padding: 3px")
-        files_div.add_color("background", "background3")
+        files_div.add_color("background", "#f7f7f7")
+        files_div.add_style("border: 3px dashed #b8e5f5")
+        files_div.add_style("border-radius: 20px 20px 20px 20px")
+        files_div.add_style("z-index: 1")
         #files_div.add_style("display: none")
 
         bgcolor = div.get_color("background3")
         bgcolor2 = div.get_color("background3", -3)
 
-        files_div.add_behavior( {
+        style_text = """
+            text-align: center;
+            margin-top: 100px;
+            color: #A0A0A0;
+            font-size: 3.0em;
+            z-index: 10;
+            """
+        background_text = DivWdg("<p style='%s'>Drag Files Here</p>" % style_text)
```

Comment:
refrain from using plain HTML. use
background_text.add_styles(style_text)... where style_text is just a one liner with ; as delimiter